test/unittests/state_transition_eip7702_test.cpp

```diff
@@ -0,0 +1,305 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
```

Comment:
Is this so old?

Reply:
I don't think it can be, will change to 2024